sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -10,13 +10,6 @@
     - `VOIP_JOIN` (Access to Calling APIs but without the authorization to start new calls)
 - Added a new API version `ApiVersion.V2023_10_01` that is now the default API version.
 
-### Breaking Changes
```

Comment:
It looks like this release has the wrong version? It should be 1.4.0, not 1.5.0.

Reply:
Corrected version to 1.4.0